apps/src/weblab2/layout/VerticalLayout.tsx

```diff
@@ -67,6 +81,38 @@ const VerticalLayout: React.FunctionComponent<LayoutProps> = ({
     appName: 'weblab2',
   });
 
+  const viewModeButtonsProps: SegmentedButtonsProps = {
+    buttons: [
```

Comment:
Do we want to hide the code/split buttons when in widget mode and viewing code is disabled?

Reply:
Oh gotcha - that makes sense. Then I think we could update [this level](https://studio.code.org/courses/original-allthethings-course/units/1/lessons/51/levels/6) later.